.circleci/config.yml

```diff
@@ -33,7 +33,8 @@ jobs:
           root: ~/
           paths:
             - project
-      - notify_slack_failure
+      - notify_slack_failure:
```

Comment:
in listing we have `slack/notify_slack_failure`. Don't we need the `slack` here?

Reply:
legacy setup and I blindly updated it 🙈 
thanks